tools/cli/commands/phan.js

```diff
@@ -662,4 +667,16 @@ export async function handler( argv ) {
 		reportStream.end();
 		console.log( `Report written to ${ argv.reportFile }` );
 	}
+	if ( ( process.exitCode & 2 ) !== 0 ) {
+		console.error(
+			chalk.red(
+				'Errors were encountered while running Phan for one or more projects! Results may not be complete.'
+			)
+		);
+		if ( ! argv.v ) {
+			console.error(
+				chalk.yellow( 'You might try running with `-v` to get more information on the failure' )
+			);
+		}
+	}
 }
```

Comment:
Should we exit this script with an exit code?

Reply:
Setting `process.exitCode` makes node exit with that code whenever it does exit.